stanford/suclass/edxapp1.yml

```diff
@@ -2,5 +2,9 @@
 ---
 - include: ../playbooks/edxapp.yml
   vars:
-    env: suclass
-    machine: app1
+    CLUSTER_INSTANCE_TYPE: 'm4.xlarge'
+    CLUSTER_NUMBER: 1
+    COMMON_DEPLOYMENT: suclass
+    nginx_sites:
```

Comment:
Just to double check, this is to override the defaults to exclude the verify site, since suclass does not do certs?

Reply:
Yup, exactly.